qiskit/circuit/quantumcircuit.py

```diff
@@ -2205,27 +2205,41 @@ def measure_active(self, inplace: bool = True) -> Optional["QuantumCircuit"]:
         else:
             return None
 
-    def measure_all(self, inplace: bool = True) -> Optional["QuantumCircuit"]:
+    def measure_all(
+        self, inplace: bool = True, add_creg: bool = True
```

Comment:
I should have thought about this before, but please could you change the name of the argument to `add_bits`?  Nowadays, the bits are the important part, and the register is just a handle to a collection of bits, so it's more important that we're adding bits.